src/Microsoft.OData.Core/UriParser/Binders/SelectExpandBinder.cs

```diff
@@ -369,15 +369,28 @@ private SelectItem GenerateExpandItem(ExpandTermToken tokenIn)
                 currentNavProp = ParseComplexTypesBeforeNavigation(currentComplexProp, ref firstNonTypeToken, pathSoFar);
             }
 
+            bool isRef = false;
+            bool isCount = false;
+            bool hasDerivedType = false;
+            PathSegmentToken derivedTypeToken = null;
+            IEdmEntityType derivedType = null;
+
+            // Handle $expand=Customer/VipCustomer
+            // The deriveTypeToken is VipCustomer
+            if (firstNonTypeToken.NextToken != null && firstNonTypeToken.NextToken.IsNamespaceOrContainerQualified())

```

Comment:
>firstNonTypeToken.NextToken.IsNamespaceOrContainerQualified() [](http://example.com/codeflow?start=55&length=61)

IsNamespaceOrContainerQualified or 
Not IsNamespaceOrContainerQualified

Reply:
@xuzhg I have updated the comment